src/caf/distribute/matrix_translation.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
```

Comment:
I think this would be useful moved to an examples folder in the repo root. Also, could you include some docstrings and type annotations